android/src/main/java/com/onegini/mobile/sdk/reactnative/OneginiSDK.kt

```diff
@@ -28,13 +28,10 @@ class OneginiSDK @Inject constructor(
     private val fingerprintAuthenticationRequestHandler: FingerprintAuthenticationRequestHandler,
     private val simpleCustomRegistrationFactory: SimpleCustomRegistrationFactory,
 ) {
-
+    var isInitialized = false
```

Comment:
SDK can only be initailized (cannot be "deinitialized"), so it might be good to make this var private and add metod that can only set it to true to prevent that in the future someone accidentaly sets that to false
```suggestion
    var isInitialized = false
        private set
```